clamav/variables.tf

```diff
@@ -25,14 +8,9 @@ variable "cf_space_name" {
   description = "cloud.gov space name (staging or prod)"
 }
 
-variable "app_name" {
-  type        = string
-  description = "base application name to be scanned by clamav"
-}
-
-variable "env" {
+variable "name" {
```

Comment:
[suggestion] this change needs a matching one within `clamav/main.tf` to update both the app's name and the route being used.

It'll be a breaking change, but I think that's ok? I'm not aware of any projects that are both using this module and also currently deploying clamav.

Reply:
Or, even better, back out these changes from this PR and include them in #3

Reply:
I'll do that, must have missed this when I split the branch the first time. ;)